hydroflow/src/lang/monotonic_map.rs

```diff
@@ -27,6 +30,7 @@ where
 impl<K, V> MonotonicMap<K, V>
 where
     K: PartialOrd,
+    V: Clear,
```

Comment:
Would be a good optimization to use clear but it looks like you aren't actually doing anything with it right now?

Reply:
This change is just restoring the code to what it was before my change, I didn't know about clear being our own trait that could be impld when I removed the trait bounds.

Reply:
Hmm ok